cuda/private/defs.bzl

```diff
@@ -0,0 +1,16 @@
+"""private defs"""
+
+def requires_cuda():
```

Comment:
Taken and adapted from https://github.com/tensorflow/runtime, there used to be rules_cuda repo in there.